lms/djangoapps/mobile_api/user_sharing_pref/urls.py

```diff
@@ -0,0 +1,14 @@
+"""
+URLs for users sharing preferences
+"""
+from django.conf.urls import patterns, url
+from .views import UserSharing
+
+urlpatterns = patterns(
+    'mobile_api.user_sharing_pref.views',
+    url(
+        r'^share_pref/$',
```

Comment:
Does this mean you just want the name to change? 
Or would you like this API endpoint to handle more then just the sharing preferences?